edb/schema/ordering.py

```diff
@@ -93,9 +93,9 @@ def linearize_delta(
 
     for _key, info in ordered:
         op = info['op']
-        if (isinstance(op, sd.AlterObject)
-                and not len(op.get_subcommands())):
-            continue
+        # if (isinstance(op, sd.AlterObject)
```

Comment:
Remove the commented code?

Reply:
Oops. Good catch.  This shouldn't be commented out.

Reply:
Then we ideally need a test for whatever that code does :)